### DISCUSSION TITLE
错误：未找到下游接口

### DISCUSSION THREAD

**Discussion by JPGC123** [Mygod/VPNHotspot#273] [Q&A]
前几天我还用的好好的，突然就变成这样子，也不知道哪的问题！卸载重装，旧版都试过了，目前只能用着临时wlan热点

**Comment by Mygod** [comment]
Export debug information?

**Comment by Mygod** [comment]
Looking at logs from #274, it seems your system does not support Wi-Fi direct, which is required for this feature to work.
